src/app/core/admin/setup-wizard/setup-wizard-button/setup-wizard-button.component.html

```diff
@@ -1,7 +1,7 @@
 @if (showSetupWizard) {
   <button
     mat-flat-button
-    routerLink="/admin/setup-wizard"
+    (click)="navigateToWizard()"
```

Comment:
Did this to not copy the url twice (although could introduce a variable also). Actually the "open in new tab" doesn't work - regardless of using routerLink or not ... (this would be nice to have though: #1217 )